guides/source/getting_started.md

```diff
@@ -72,92 +72,63 @@ By following along with this guide, you'll create a Rails project called
 your prompt will look something like `C:\source_code>`.
 
-### Installing Rails
-
-Before you install Rails, you should check to make sure that your system has the
-proper prerequisites installed. These include:
-
-* Ruby
-* SQLite3
-
-#### Installing Ruby
-
-Open up a command line prompt. On macOS open Terminal.app; on Windows choose
-"Run" from your Start menu and type `cmd.exe`. Any commands prefaced with a
-dollar sign `$` should be run in the command line. Verify that you have a
-current version of Ruby installed:
-
-```bash
-$ ruby --version
-ruby 3.1.0
-```
-
-Rails requires Ruby version 3.1.0 or later. It is preferred to use the latest Ruby version.
-If the version number returned is less than that number (such as 2.3.7, or 1.8.7),
-you'll need to install a fresh copy of Ruby.
+### Installing rails-new
 
-To install Rails on Windows, you'll first need to install [Ruby Installer](https://rubyinstaller.org/).
+This guide utilizes the `rails-new` tool to create a Rails app on your machine. We will
+then run our application in a Dev Container using VSCode. Both rails-new and the
+Dev Container run in a containerized environment using Docker. So we will need to install
+VSCode and Docker as well as `rails-new`.
 
-For more installation methods for most Operating Systems take a look at
-[ruby-lang.org](https://www.ruby-lang.org/en/documentation/installation/).
+#### Installing Docker
 
-#### Installing SQLite3
+Dev Containers are run using Docker. You can install Docker by following the
+installation instructions for your operating system in the [Docker docs](https://docs.docker.com/desktop/).
 
-You will also need an installation of the SQLite3 database.
-Many popular UNIX-like OSes ship with an acceptable version of SQLite3.
-Others can find installation instructions at the [SQLite3 website](https://www.sqlite.org).
+Once Docker has been installed, open it from your application folder to begin running
+the Docker engine on your machine.
 
-Verify that it is correctly installed and in your load `PATH`:
-
-```bash
-$ sqlite3 --version
-```
-
-The program should report its version.
-
-#### Installing Rails
-
-To install Rails, use the `gem install` command provided by RubyGems:
-
-```bash
-$ gem install rails
-```
+#### Installing VSCode
 
-To verify that you have everything installed correctly, you should be able to
-run the following in a new terminal:
+The easiest way of working with Dev Containers is using the VSCode editor. You can
+install VSCode by downloading it from [the website](https://code.visualstudio.com/).
 
-```bash
-$ rails --version
-Rails 7.2.0
-```
+#### Installing rails-new
 
-If it says something like "Rails 7.2.0", you are ready to continue.
+`rails-new` is a tool which can generate a new Rails application without needing
+Ruby or Rails installed on your machine. To install rails-new, follow the installation
+instructions [in the readme](https://github.com/rails/rails-new?tab=readme-ov-file#installation).
```

Comment:
@andrewn617 I was just trying to install this locally and I don't see any installation instructions in the readme for `rails-new`?

Reply:
I was chatting with Rafael about this today and it's next on my todo list. I'll have a PR open for it in the next 24 hours.